src/components/RelocationBox/RelocationBox.jsx

```diff
@@ -184,6 +245,30 @@ const RelocationBox = ({
             </h3>
             {/* Box picture */}
             <AccordionPanel pb={4} className={styles['accordion-panel']}>
+              <div className={styles['review-header']}>
+                <h4>{status === 'pending changes' ? 'Reviewed by FYA Admin' : ''}</h4>
+                {status === 'pending changes' && (
+                  <button
+                    type="button"
+                    style={true ? {} : { visibility: 'hidden' }}
+                    onClick={() => {
+                      setEditPendingChangesState(!editPendingChangesState || !isValid);
+                    }}
+                  >
+                    {editPendingChangesState ? (
+                      <RiCheckFill
+                        color="#38a169"
+                        size={20}
+                        onClick={() => {
+                          handleSubmit(onSubmit)();
+                        }}
```

Comment:
I think you can just put `onClick={handleSubmit(onSubmit)}` here? This will make it look a bit cleaner

Reply:
fixed :)